frontend/src/components/management/WorkflowAssignment.tsx

```diff
@@ -101,8 +129,19 @@ export function WorkflowAssignment({ fixedFilter }: LabworkSamplesProps) {
         setOpen(false)
     }, [])
 
+    console.info(fixedFilters)
     return (
         <>
+            {Object.keys(wholeFilters).length > 0 && (
+                <>
+                    <b>Fixed Filters:</b>
+                    <ul>
+                        {Object.values(fixedFilters).map((setting, index) => (
+                            <li key={setting.description?.key ?? index}>{`${setting.description?.label}: ${setting.value}`}</li>
```

Comment:
Not sure about the retro looking list of fixed filters... The filter is listed in the table header but cannot be cleared. if you remove it from the column the exact is stuck and the filter does not accept other values ... I think fixed filter were meant for components that cannot take other values for a given column ... (Exemple project column for ProjectAssociatedSamples). In our case one could decide to clear the filter and input something else ... It if more of a default filter than a fixed one.

Reply:
Hmm... maybe I should hide the column instead?